### PR TITLE
bpo-34682: Wording and grammatical changes to the doc(https://docs.python.org/3)

### PR DESCRIPTION
**Pull request by divyag9**
https://bugs.python.org/issue34682

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

File: Doc/reference/expressions.rst [hunk 1]
```diff
@@ -523,7 +523,7 @@ is already executing raises a :exc:`ValueError` exception.
 .. method:: generator.__next__()
 
    Starts the execution of a generator function or resumes it at the last
-   executed yield expression.  When a generator function is resumed with a
```
Comment (ericvsmith) [inline Doc/reference/expressions.rst]: I think the original "a" is correct here, although I realize there's some wiggle room.

File: Doc/reference/expressions.rst [hunk 2]
```diff
@@ -1594,8 +1594,8 @@ The operator :keyword:`not in` is defined to have the inverse true value of
 Identity comparisons
 --------------------
 
-The operators :keyword:`is` and :keyword:`is not` test for object identity: ``x
-is y`` is true if and only if *x* and *y* are the same object.  Object identity
+The operators :keyword:`is` and :keyword:`is not` test for object's identity: ``x
+is y`` is true if and only if *x* and *y* are the same object.  Object's identity
```
Comment (ericvsmith) [inline Doc/reference/expressions.rst]: And same here: "An object's identity ..."

File: Doc/reference/expressions.rst [hunk 3]
```diff
@@ -1594,8 +1594,8 @@ The operator :keyword:`not in` is defined to have the inverse true value of
 Identity comparisons
 --------------------
 
-The operators :keyword:`is` and :keyword:`is not` test for object identity: ``x
-is y`` is true if and only if *x* and *y* are the same object.  Object identity
+The operators :keyword:`is` and :keyword:`is not` test for object's identity: ``x
```
Comment (ericvsmith) [inline Doc/reference/expressions.rst]: I would make this "test for an object's identity" (adding "an").

**Review by ericvsmith — CHANGES_REQUESTED** [review]
Other than my comments, this looks good. Thanks!

**Comment by divyag9** [comment]
I have made the requested changes; please review again

**Review by ericvsmith — APPROVED** [review]
Looks good. Thanks!

**Comment by csabella** [comment]
@divyag9 Congratulations on getting your PR merged!  :tada: :tada:

**Comment by divyag9** [comment]
> @divyag9 Congratulations on getting your PR merged! 🎉 🎉

Thanks Cheryl! Thanks for all your help

**Pull request merged**